docs/api/api_alternative.md

```diff
@@ -0,0 +1,289 @@
+| - | - | - |
+| [BLE](../apis/ble.html) | &#10004; | &#10003; (can be manually enabled) |
+| [GAP](../apis/gap.html) | &#10004; | &#10003; (can be manually enabled) |
+| [GattClient](../apis/gattclient.html) | &#10004; | &#10003; (can be manually enabled) |
+| [GattServer](../apis/gattserver.html) | &#10004; | &#10003; (can be manually enabled) |
+| [SecurityManager](../apis/SecurityManager.html) | &#10004; | &#10003; (can be manually enabled) |
+| [BatteryService](../apis/batteryservice.html) | &#10004; | &#10003; (can be manually enabled) |
+| [HeartRateService](../apis/heartrateservice.html) | &#10004; | &#10003; (can be manually enabled) |
+
+### NFC
+
+You can use Near-Field Communication (NFC), a short-range radio technology, for use cases such as contactless payments, access control and device pairing.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [NFCController](../apis/nfccontroller.html) | &#10004; |&#10003; (can be manually enabled) |
+| [NFC EEPROM](../apis/nfc-eeprom.html) | &#10004; | &#10003; (can be manually enabled) |
+
+### LoRaWAN
+
+Arm Mbed OS provides a native network stack for LoRaWAN. LoRaWAN is a technology designed for low-power battery-powered devices. These devices operate in an unlicensed spectrum, creating high density wide-area networks.
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [LoRaWANInterface](../apis/lorawaninterface.html) | &#10004; | |
+| [LoRaRadio](../apis/nfc-loraradio.html) | &#10004; | |
+
+
+## Security
+
+With [Arm Mbed TLS](security.html), a comprehensive SSL/TLS solution, you can include cryptographic and SSL/TLS capabilities in your code.
+
+
+| API | Full profile | Bare metal profile |
+| - | - | - |
+| [PSA initial attestation](../apis/security-apis.html) | &#10004; | |
+| [PSA lifecycle](../apis/psa-lifecycle.html) | &#10004; |  |
+| [Mbed Crypto](../apis/mbed-crypto.html) | &#10004; |&#10003; (can be manually enabled) |
+| [TLS](../apis/tls.html) | &#10004; | &#10003; (can be manually enabled) |
+| [DeviceKey](../apis/devicekey.html) | &#10004; | &#10003; (can be manually enabled) |
+
+
+
+## Deprecated APIs: moving from Mbed OS 5 to 6
+
+If you're moving your program from Mbed OS 5 to 6, you will need to replace deprecated APIs:
+
+| Deprecated API | Replaced by |
+| - | - |
+| `Serial` |`printf` and `puts` to access the console. <br> `BufferedSerial` for blocking applications.<br> `UnbufferedSerial` for bypassing locks in IRQ or short of RAM. |
+| `RawSerial` | `UnbufferedSerial` |
+| `UARTSerial` | `BufferedSerial` |
+| `Ethernet` | `EthInterface` to get an Ethernet object. <br> `NetworkInterface` to get an instance of an appropriate network interface (WiFi or Ethernet). |
+| BLE services: `iBeacon`, `UARTService`, `URIBeaconConfigService` | No replacement available |
+| `TCPServer` | `TCPSocket` |
+| `NVStore` | `KVStore` |
+| `FunctionPointerArg1<R, A>` & `FunctionPointerArg1<R, void>` | `Callback<R(A)>` & `Callback<void()>` |
```

Comment:
I provided the list for completeness but I would list only is there was a .md file for the API. Can you check if this one and those below were mentioned in 5.15?

Reply:
"FunctionPointerArg1" not in the docs
"RtosTimer" mentioned in Tickless and Execution
"InterruptManager" not in the docs
"CallChain" not in the docs

I'm more concerned with "will they be able to understand why their code doesn't compile" than "explicitly had a page"

Reply:
I propose to remove "FunctionPointerArg1" and keep the others then.